templates/github/.github/workflows/fips.yml.j2

```diff
@@ -0,0 +1,68 @@
+{% include 'header.j2' %}
+---
+name: Pulp FIPS Nightly CI
+on:
+  {% if fips_test -%}
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # runs at 4:00 UTC daily
+    - cron: '00 4 * * *'
+  {%- else -%}
+  workflow_dispatch:
+  {%- endif %}
+
+jobs:
+  vagrant:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - test_type: pulp3-source-centos8-fips
+          - test_type: pulp3-source-centos7-fips
+    steps:
+      - uses: actions/checkout@v2.3.1
+        with:
+          submodules: true
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Ansible
+        run: |
+          pip3 install -U pip
+          pip3 install "ansible>2.9,<3"
+          ansible --version
+      - name: Pulp Installer
+        run: |
+          cd ..
+          git clone --depth=1 https://github.com/pulp/pulp_installer
+      - name: Install Vagrant
+        shell: bash
+        run: |
+          cd ../pulp_installer/
+          git submodule update --init
+          {%- if plugin_name in ['pulp_file', 'pulpcore'] %}
+          sed -i "/{{ plugin_name }}/d" .github/workflows/scripts/install.sh
+          {%- endif %}
+          .github/workflows/scripts/install.sh {{ "${{ matrix.test_type }}" }}
+      - name: Run Vagrant
+        shell: bash
+        run: |
+          cd ../pulp_installer
+          {% if plugin_name not in ['pulp_file', 'pulpcore'] -%}
+          sed -e "s/pulp-file/{{ plugin_dash }}/g" -e "s/pulp_file/{{ plugin_name }}/g" -e "/pulp_source_dir/a pulp_webserver_disable_https: true" example.dev-config.yml > local.dev-config.yml
```

Comment:
:+1:
Maybe in a next iteration, we should just provide the proper dev-config in the assets directory. It can use the additional_plugins variable at templating to determine what to install.